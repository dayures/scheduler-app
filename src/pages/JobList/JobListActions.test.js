import React from 'react'
import { shallow } from 'enzyme'
import JobListActions from './JobListActions'

describe('<JobListActions>', () => {
    it('renders correctly for configurable jobs', () => {
        const wrapper = shallow(<JobListActions id="1" configurable />)

        expect(wrapper).toMatchSnapshot()
    })

    it('renders correctly for non configurable jobs', () => {
        const wrapper = shallow(<JobListActions id="1" />)

        expect(wrapper).toMatchSnapshot()
    })
})
