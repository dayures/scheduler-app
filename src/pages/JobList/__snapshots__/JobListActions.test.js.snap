// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<JobListActions> renders correctly for configurable jobs 1`] = `
<DropdownButton
  component={
    <FlyoutMenu
      dataTest="dhis2-uicore-menu"
      maxHeight="auto"
      maxWidth="380px"
    >
      <EditJobMenuItem
        id="1"
      />
      <RunJobMenuItem
        id="1"
      />
      <DeleteJobMenuItem
        id="1"
      />
    </FlyoutMenu>
  }
  dataTest="dhis2-uicore-dropdownbutton"
  small={true}
>
  Actions
</DropdownButton>
`;

exports[`<JobListActions> renders correctly for non configurable jobs 1`] = `
<DropdownButton
  component={
    <FlyoutMenu
      dataTest="dhis2-uicore-menu"
      maxHeight="auto"
      maxWidth="380px"
    >
      <RunJobMenuItem
        id="1"
      />
      <DeleteJobMenuItem
        id="1"
      />
    </FlyoutMenu>
  }
  dataTest="dhis2-uicore-dropdownbutton"
  small={true}
>
  Actions
</DropdownButton>
`;
